scripts/lb-wrapper

```diff
@@ -4,16 +4,16 @@ LOG_FILE="/var/log/xklb.log"
 XKLB_EXECUTABLE="${XKLB_EXECUTABLE:-lb}"
 VERBOSITY="-vv"
 TMP_DOWNLOADS_DIR="/library/downloads/calibre-web"
-SURVEY_DB_FILE="${TMP_DOWNLOADS_DIR}/survey.db"
+XKLB_DB_FILE="/library/calibre-web/xklb-metadata.db"
```

Comment:
maybe some of these could be env vars instead of defined across BASH and python

Reply:
> maybe some of these could be env vars instead of defined across BASH and python

Great idea 💡

@deldesir: Maybe in a separate PR?